README.md

```diff
@@ -538,6 +538,37 @@ en:
 Of course, this is just an example. Pundit is agnostic as to how you implement
 your error messaging.
 
+## Deep error messages customisation
+
+If you have different authorization deny reasons you want to inform user with descriptive message then:
```

Comment:
"deny reason" feels like a new bit of terminology: I'd prefer to avoid that if possible. Consider:
```suggestion
If there are multiple reasons that authorization can be denied, you can show different messages by raising exceptions in your policy:
```